gpu/install_gpu_driver.sh

```diff
@@ -622,13 +624,13 @@ function install_nvidia_nccl() {
     local local_tarball="${workdir}/${build_tarball}"
     local gcs_tarball="${pkg_bucket}/nvidia/nccl/${_shortname}/${build_tarball}"
 
-    if [[ "$(hostname -s)" =~ ^test && "$(nproc)" < 32 ]] ; then
+    if [[ "$(hostname -s)" =~ ^test-gpu && "$(nproc)" < 32 ]] ; then
```

Comment:
internal test clusters start with test, but not test-gpu ; exclude the internal test suite from the following logic by using github cluster format regex